bridge_adaptivity/module/views.py

```diff
@@ -646,11 +685,14 @@ def preview_collection(request, slug):
         }
         for pos, a in enumerate(get_list_or_404(Activity, collection__slug=slug), start=1)
     ]
-
     return render(
         request,
         template_name="module/sequence_preview.html",
-        context={'activities': acitvities}
+        context={
+            'activities': acitvities,
+            'back_url': f"{reverse('module:collection-detail', kwargs={'pk':Collection.objects.get(slug=slug).pk})}"
+                        f"?back_url={request.GET.get('back_url')}"
```

Comment:
incorrect indentation

Reply:
If you meant it should be like this:
```python
        context={
            'activities': acitvities,
            'back_url': f"{reverse('module:collection-detail', kwargs{'pk':Collection.objects.get(slug=slug).pk})}"
                f"?back_url={request.GET.get('back_url')}"
        }
```
The flake8 will rase error:
```
flake8 runtests: commands[0] | flake8
./bridge_adaptivity/module/views.py:689:17: E131 continuation line unaligned for hanging indent
                f"?back_url={request.GET.get('back_url')}"
                ^
ERROR: InvocationError for command '/Volumes/edx/projects/aloci/.tox/flake8/bin/flake8' (exited with code 1)
```

But if I run flake8 without changes, I won't have got any error from flake:
```
flake8 runtests: commands[0] | flake8
______________________________________________________________________________________ summary ______________________________________________________________________________________
  flake8: commands succeeded
  congratulations :)
```

if I didn't undestand you. Please write more information about it.